code/_onclick/hud/alien.dm

```diff
@@ -71,78 +81,76 @@
 	using.icon_state = "swap_2"
 	using.screen_loc = ui_swaphand2
 	using.layer = 19
-	adding += using
+	static_inventory += using
 
-	using = new /obj/screen/act_intent()
-	using.icon = 'icons/mob/screen_alien.dmi'
+	using = new /obj/screen/act_intent/alien()
 	using.icon_state = mymob.a_intent
-	using.screen_loc = ui_movi
-	adding += using
+	static_inventory += using
 	action_intent = using
 
 	if(istype(mymob, /mob/living/carbon/alien/humanoid/hunter))
 		var/mob/living/carbon/alien/humanoid/hunter/H = mymob
 		H.leap_icon = new /obj/screen/alien/leap()
 		H.leap_icon.screen_loc = ui_alien_storage_r
-		adding += H.leap_icon
+		static_inventory += H.leap_icon
 
 	using = new /obj/screen/drop()
 	using.icon = 'icons/mob/screen_alien.dmi'
 	using.screen_loc = ui_drop_throw
-	adding += using
+	static_inventory += using
 
 	using = new /obj/screen/resist()
 	using.icon = 'icons/mob/screen_alien.dmi'
 	using.screen_loc = ui_pull_resist
-	adding += using
+	hotkeybuttons += using
 
-	mymob.throw_icon = new /obj/screen/throw_catch()
-	mymob.throw_icon.icon = 'icons/mob/screen_alien.dmi'
-	mymob.throw_icon.screen_loc = ui_drop_throw

```

Comment:
I moved many obj/screen objects var (like throw_icon, pull_icon, healths, internals) from the mob to the hud, because it makes more sense.